x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/FieldAttributeTests.java

```diff
@@ -182,7 +182,7 @@ public void testDottedFieldPathTypo() {
     public void testStarExpansionExcludesObjectAndUnsupportedTypes() {
```

Comment:
I think you need to look at `testUnsignedLongVersionCompatibility` as a test example for `version`. What do you think @bpintea?